brainscore_core/submission/database_models.py

```diff
@@ -5,9 +5,13 @@
 
 
 class PeeweeBase(PeeweeModel):
+    @classmethod
+    def set_schema(cls, schema_name):
+        for model in cls.__subclasses__():
+            model._meta.schema = schema_name
+    
     class Meta:
         database = database_proxy
-        schema = 'public'
```

Comment:
The default schema for our database is already 'public'! So if no schema is set, this will automatically default to public. If you think it'd make it more clear I can add `schema_name='public'` to the header of `set_schema`. Thoughts?

Reply:
I see. If that's the case, looks good to me. No need to add 'public' to the header.